source/components/text-input/text-input.less

```diff
@@ -9,8 +8,13 @@ textarea {
   padding: 8px;
   width: 100%;
 
+  textarea& {
+    min-height: 5.75rem; // ~92px
+    resize: vertical;
```

Comment:
I know of no scenario where a user would need to resize a textarea horizontally.